rrfs-test/validated_yamls/templates/obtype_config/ztd_ukmo.yaml

```diff
@@ -0,0 +1,70 @@
+     - obs space:
+         name: Groundgnss  
```

Comment:
Could this be named something like:
```yaml
         name: gnss_zenithTotalDelay_123
```

To try to match what is in the mesonet and metar yamls. Does ztd have a observation type number? For example mesonet would be:
```yaml
         name: msonet_airTemperature_188
```

Reply:
updated.